client/src/components/_common/Button/Button.module.css

```diff
@@ -0,0 +1,56 @@
+.c-button {
+  composes: c-button from '../../../styles/components/c-button.css';
+}
+.c-button:focus,
+[class*='c-button--']:focus {
+  outline: none;
+}
```

Comment:
**📌  Common Practice**
Usually, we use `.root` or `container` for the base element. For details, see [another comment].